File: check-if-word-equals-summation-of-two-words.ts
function isSumEqual(
  firstWord: string,
  secondWord: string,
  targetWord: string
): boolean {
  function convert(word) {
    return +word
      .split("")
      .map((c) => c.charCodeAt(0) - 97)
      .join("");
  }

  return convert(firstWord) + convert(secondWord) === convert(targetWord);
}

File: check-if-word-equals-summation-of-two-words.py
class Solution:
    def getNumericalValue(self, word) -> str:
        return int(''.join([str(ord(c) - 97) for c in word]))

    def isSumEqual(self, firstWord: str, secondWord: str, targetWord: str) -> bool:
        return self.getNumericalValue(firstWord) + self.getNumericalValue(secondWord) == self.getNumericalValue(targetWord)
